samples/highcharts/demo/area-race-chart/demo.js

```diff
@@ -0,0 +1,321 @@
+const btn = document.getElementById('play-pause-button'),
+    input = document.getElementById('play-range'),
+    startYear = 1973,
+    endYear = 2021;
+
+// General helper functions
+const arrToAssociative = arr => {
+    const tmp = {};
+    arr.forEach(item => {
+        if (item[1] !== null) {
+            tmp[item[0]] = item[1];
+        }
+    });
+
+    return tmp;
+};
+
+function getSubtitle() {
+    return `<span style='font-size: 60px'>${input.value}</span>`;
+}
+
+const formatRevenue = [];
+
+const chart = Highcharts.chart('container', {
+    chart: {
+        events: {
+            // Some annotation labels need to be rotated to make room
+            load: function () {
+                const labels = this.annotations[0].labels;
+                labels
+                    .find(a => a.options.id === 'vinyl-label')
+                    .graphic.attr({
+                        rotation: -20
+                    });
+                labels
+                    .find(a => a.options.id === 'cassettes-label')
+                    .graphic.attr({
+                        rotation: 20
+                    });
+            }
+        },
+        type: 'area',
+        marginTop: 100,
+        animation: {
+            duration: 700,
+            easing: function (t) {
+                return t;
+            }
+        }
+    },
+    title: {
+        text: 'Music revenue race chart'
+    },
+    subtitle: {
+        useHTML: true,
```

Comment:
Why useHTML? It should only be used if there is formatting that we can't do in SVG.